### PR TITLE
Milestone: Style bookstore

### PR DESCRIPTION
**Pull request by Gleennkar**
## Added style to the bookstore app
- Check linter errors
- Added style 
- Updated my README.md

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ function Books() {
 
   return (
     <>
-      <div className="book-container">
+      <div className="container-fluid d-flex-column w-80 mt-5 book-container">
         {Object.keys(library.Booklist).map((bookItem) => (
 
-          <div className="book" key={uuidv4()}>
+          <div className="d-flex mt-3 book" key={uuidv4()}>
```
Comment (victorokoroji) [inline src/components/Books.js]: - [ ] Kindly ensure that your bookstore book styling exactly matches the design you are given. Kindly add all the missing parts such as the progress bar 👍🏽.

Zeplin design 👇🏽 
image

Your design 👇🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Gleennkar, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅  
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (tufoinnkuo10) [inline src/components/Books.js]: - [ ] like the previous reviewer noted, Kindly add all the missing parts such as the progress bar so it matches the Zeplin design 👍🏽.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Gleennkar 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.css [hunk 2]
```diff
@@ -13,3 +13,156 @@ body {
...
+}
+
+.book-container {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em;
+  border: 1px solid #e8e8e8;
+  border-radius: 4px;
+}
```
Comment (rotimiazeez) [inline src/index.css]: - Please note that your design does not match the original Zeplin design. Kindly make sure your design looks similar (font, color, margin, etc.) before submitting again. 😉
- Original Design
git
- Your Design
Screenshot from 2022-11-13 06-58-06

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Gleennkar  😉, 
## Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by bellom** [comment]
@Gleennkar, 

### This is a manual review

According to the project requirement, **your design should match the design on Zeplin.**
image

As stated in the project requirement, **kindly use the information detailed in Zeplin to style your website.**
For example: look at the properties for `Action` in the screenshot below;
image


Your design:
image


I hope this is helpful.

**Review by julie-ify — APPROVED** [review]
Hi @Gleennkar,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**